Scripts/Editor/AssetBundleWindow.cs

```diff
@@ -51,6 +56,7 @@ public class AssetBundleWindow : EditorWindow
         public bool disableWriteTypeTree = false;
         public bool deterministicAssetBundle = false;
         public bool uncompressedAssetBundle = false;
+        public bool removeVariantText = false;
```

Comment:
This feature only works sometimes but it is a checkbox to try and remove the end-capping variant keyword that is put at the end of some prefabs.